djangogirls/settings.py

```diff
@@ -276,3 +276,15 @@
 GAPPS_ADMIN_SDK_SCOPES = 'https://www.googleapis.com/auth/admin.directory.user'
 GAPPS_PRIVATE_KEY_ID = os.environ.get('GAPPS_PRIVATE_KEY_ID', '')
 GAPPS_PRIVATE_KEY = os.environ.get('GAPPS_PRIVATE_KEY', '')
+
+DATABASES = {
```

Comment:
We need to remove this bit - it contains your local password and we have a proper db configuration earlier in the file.